eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -82,6 +82,12 @@ stages:
               vmImage: MMSUbuntu18.04
 
             steps:
+              - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+                parameters:
+                  Paths:
+                    - '${{ parameters.ServiceDirectory }}'
+                    - '**/*.xml'
```

Comment:
Why are the xml files needed here? Is it because the version update checks them? Is that all it checks?

Reply:
I have the xml files included for `set_versions.py` below. It's possible it has more minimal file dependencies in this case, but in other places I've seen it require the xml files.